templates/tinted-vim.mustache

```diff
@@ -277,10 +305,10 @@ hi! link SnippetTabstop Visual
 call <sid>hi('SpecialKey',     s:gui03, '', s:cterm03, '', '', '')
 
 " Spell
-call <sid>hi('SpellBad',       '', '', s:ctermbg, s:cterm12, 'undercurl', s:gui08)
-call <sid>hi('SpellLocal',     '', '', s:ctermbg, s:cterm15, 'undercurl', s:gui15)
-call <sid>hi('SpellCap',       '', '', s:ctermbg, s:cterm16, 'undercurl', s:gui16)
-call <sid>hi('SpellRare',      '', '', s:ctermbg, s:cterm0E, 'undercurl', s:gui0E)
+call <sid>hi('SpellBad',       '', '', s:ctermbg, s:cterm12, s:attrs.undercurl, s:gui08)
+call <sid>hi('SpellLocal',     '', '', s:ctermbg, s:cterm15, s:attrs.undercurl, s:gui15)
+call <sid>hi('SpellCap',       '', '', s:ctermbg, s:cterm16, s:attrs.undercurl, s:gui16)
+call <sid>hi('SpellRare',      '', '', s:ctermbg, s:cterm0E, s:attrs.undercurl, s:gui0E)
```

Comment:
Please keep the string typed parameter for the attributes and handle the logic for enabling and disabling in the `TintedHi` function.

Reply:
[This should address ](https://github.com/tinted-theming/tinted-vim/pull/102/commits/0cf0a52f28e95350cc31fd169868898985429ff3)